sway-lib-std/src/chain.sw

```diff
@@ -4,6 +4,12 @@ dep chain/auth;
 use ::panic::panic;
 
 // When generics land, these will be generic.
+pub fn log_b256(value: b256) {
```

Comment:
This needs a test

Reply:
Yeah, will write in a second, just wanted to get this started.